examples/handling-buckets/create-bucket.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+include __DIR__.'/../../vendor/autoload.php';
+
+use Supabase\Storage\StorageClient;
+
+$api_key = getenv('API_KEY');;
```

Comment:
Let's make sure we update all of the examples to use this pattern.

Reply:
Done!